src/wallet/wallet.cpp

```diff
@@ -5152,6 +5161,7 @@ void CWallet::SetNull()
     fBackupMints = false;
 
     // Stake Settings
+    pStakerStatus = new CStakerStatus();
```

Comment:
Not fan of this line. Prefer a:

```
pStakerStatus = (pStakerStatus) ? pStakerStatus->clear() : new CStakerStatus();
```

To prevent anyone in the future calling `SetNull` twice and ending up with a non deleted allocation.

Reply:
:+1: nice idea